src/main/frontend/db/rules.cljs

```diff
@@ -30,3 +30,18 @@
     ;;            (not-join [?e ?v]
     ;;                      [?e ?a ?v]))]
     ])
+
+(def query-dsl-rules
```

Comment:
We can make most of the query-dsl functionality available as rules to empower users to use them in other contexts :)